python/lsst/ctrl/mpexec/cmdLineFwk.py

```diff
@@ -511,7 +512,8 @@ def configLog(longlog, logLevels):
             message_fmt = "%c %p: %m%n"
 
         # global logging config
-        lsst.log.configure_prop(_LOG_PROP.format(message_fmt))
+        if ("LSST_LOG_CONFIG" not in os.environ) or (not os.path.isfile(os.environ["LSST_LOG_CONFIG"])):
```

Comment:
Could probably shorten this to:

    if not os.path.isfile(os.environ.get("LSST_LOG_CONFIG", "")):